### ISSUE TITLE
Version : 5.0.4deb2 - 2FA missing composer

### ISSUE THREAD

**Issue by stratege1401** [phpmyadmin/phpmyadmin#16890]
Hello,

PhpMyAdmin 4:5.0.4+dfsg2-2
Server :  Debian Bullseye RC1 5.10.28-1 ( testing purpose )
Server version: Apache/2.4.46 (Debian)
Server built:   2021-01-11T10:58:23
PHP 7.4.15 (cli) (built: Feb 20 2021 09:45:56) ( NTS )
Server version: 10.5.9-MariaDB-1 

PhpMyAdmin install via apt-get

Option using Apache2 works fine.
No database problems.

Just missing 3 composer for F2A:
- pragmarx/google2fa-qrcode
- bacon/bacon-qr-code
- samyoul/u2f-php-server

Same problem, different package: #15845

**Comment by williamdes** [comment]
Hi @stratege1401 

You report duplicates #16779 where you will find more information, but for now installing manually is the best option.

This situation is blocked by things out of the scope of phpMyAdmin, that said I do my best to get them solved.
I hope to have the 2FA packages working when 5.1 will be into Debian

**Comment by williamdes** [comment]
Duplicate of #16779
